### DISCUSSION TITLE
Welcome to Arduino-CMake-Toolchain Discussions!

### DISCUSSION THREAD

**Discussion by technyon** [technyon/Arduino-CMake-Toolchain#4] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by TobyChaloner** [comment]
Hi, I'm a software engineer, interested in creating a number of simple Arduino / ESP controllers.  My motivation for tool smithing is that if I get stuck, someone else may also.
